overlord/snapstate/storehelpers.go

```diff
@@ -558,7 +558,7 @@ func storeUpdatePlanCore(
 	}
 
 	for _, name := range localAmends {
-		hasLocalRevision[allSnaps[name]] = updates[name].RevOpts
+		hasLocalRevision[name] = allSnaps[name]
```

Comment:
is there a deep reason to this change or just a simplification?

Reply:
Just a simplification and works better with upcoming changes.